ansible/group_vars/production/resources.yml

```
@@ -91,3 +91,7 @@ AMQP_FILE_PREVIEW_CPU_LIMITS: "4000m"
 AMQP_FILE_PREVIEW_CPU_REQUESTS: "100m"
 AMQP_FILE_PREVIEW_MEMORY_LIMITS: "4Gi"
 AMQP_FILE_PREVIEW_MEMORY_REQUESTS: "4Gi"
+TLDRAW_CLIENT_CPU_LIMITS: "500m"
+TLDRAW_CLIENT_MEMORY_LIMITS: "2Gi"
+TLDRAW_CLIENT_CPU_REQUESTS: "500m"
+TLDRAW_CLIENT_MEMORY_REQUESTS: "2Gi"
```

I think it better to make the CPU Usage more skalabel vorm 1/10 to 1 CPU Core per second
```suggestion
TLDRAW_CLIENT_CPU_LIMITS: "100m"
TLDRAW_CLIENT_MEMORY_LIMITS: "2Gi"
TLDRAW_CLIENT_CPU_REQUESTS: "1000m"
TLDRAW_CLIENT_MEMORY_REQUESTS: "2Gi"
```